### ISSUE TITLE
Could not get owner of name 'org.guake3.RemoteControl': no such name

### ISSUE THREAD

**Issue by ludovicaelbrecht** [Guake/guake#1511]
- **Guake version**: git latest master, a85c905459755bdf49e9a864d6ce5a069672434c
  - **System**: Fedora 29, gnome 3.30 installed, but running KDE

I removed the version of guake that was installed through the Fedora repo (dnf remove guake). Then I did this:
```
$ git pull
$ ./scripts/bootstrap-dev-fedora.sh
$ make
$ sudo make install
```

Everything works fine so far. But running guake gives me this:
```
$ guake
Guake not running, starting it
Traceback (most recent call last):
  File "/usr/lib64/python3.7/site-packages/dbus/bus.py", line 175, in activate_name_owner
    return self.get_name_owner(bus_name)
  File "/usr/lib64/python3.7/site-packages/dbus/bus.py", line 361, in get_name_owner
    's', (bus_name,), **keywords)
  File "/usr/lib64/python3.7/site-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NameHasNoOwner: Could not get owner of name 'org.guake3.RemoteControl': no such name

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/guake/main.py", line 319, in main
    remote_object = bus.get_object(DBUS_NAME, DBUS_PATH)
  File "/usr/lib64/python3.7/site-packages/dbus/bus.py", line 241, in get_object
    follow_name_owner_changes=follow_name_owner_changes)
  File "/usr/lib64/python3.7/site-packages/dbus/proxies.py", line 248, in __init__
    self._named_service = conn.activate_name_owner(bus_name)
  File "/usr/lib64/python3.7/site-packages/dbus/bus.py", line 180, in activate_name_owner
    self.start_service_by_name(bus_name)
  File "/usr/lib64/python3.7/site-packages/dbus/bus.py", line 278, in start_service_by_name
    'su', (bus_name, flags)))
  File "/usr/lib64/python3.7/site-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name org.guake3.RemoteControl was not provided by any .service files

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/guake", line 10, in <module>
    sys.exit(exec_main())
  File "/usr/local/lib/python3.7/site-packages/guake/main.py", line 448, in exec_main
    if not main():
  File "/usr/local/lib/python3.7/site-packages/guake/main.py", line 333, in main
    from guake.guake_app import Guake
  File "/usr/local/lib/python3.7/site-packages/guake/guake_app.py", line 69, in <module>
    from guake.notebook import NotebookManager
  File "/usr/local/lib/python3.7/site-packages/guake/notebook.py", line 35, in <module>
    gi.require_version('Wnck', '3.0')
  File "/usr/lib64/python3.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Wnck not available
```

I noticed a number of similar issues were already created on this, but none seem to apply to me. Issue #1185 mentions the guake schema which needs to be installed, but that seems OK. Here's what I have (after doing an updatedb):
```
$ locate org.guake.gschema.xml
/usr/local/share/glib-2.0/schemas/org.guake.gschema.xml
/usr/local/share/guake/data/org.guake.gschema.xml
/usr/src/local-builds/guake/data/org.guake.gschema.xml
```

Any suggestions as to what could be wrong?

Thanks,
Ludovic

**Comment by gsemet** [comment]
you may need to remove old version of guake and clean these .gschema.xml files to avoid conflicts (and need a restart)

**Comment by freefcw** [comment]
> you may need to remove old version of guake and clean these .gschema.xml files to avoid conflicts (and need a restart)

hello, i meet the same problem,  3b1f811104157bc4a7ecd6c0ba450c3887070e1c

I have remove all old guake packages, and restart my machine, won't solve this problem. 

the setup follow document: https://guake.readthedocs.io/en/latest/user/installing.html#install-from-source

**Comment by freefcw** [comment]
I have install gir1.2-wnck-3.0 package solve this problem.

```apt install gir1.2-wnck-3.0```

**Comment by mlouielu** [comment]
Is because other exception raised so given `org.guake3.RemoteContro: no such name` exception.

At the last line, it shows that it didn't find `Wnck`:

```
  File "/usr/lib64/python3.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Wnck not available
```

As @freefcw mention, install `gir1.2-wnck-3.0` can solve this problem.

**Comment by themilkman** [comment]
> I have install gir1.2-wnck-3.0 package solve this problem.
> 
> `apt install gir1.2-wnck-3.0`

Thanks for sharing this, solved my problem in an just upgraded Kubuntu 19.10 with Guake 3.6.3
